CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ N/A
 
 ## Release Highlights
 
+- Added support for semicolons in query strings.
```

Comment:
This needs to be moved into the changes since v7.5.1 section and follow the pattern there

Reply:
@JoelSpeed I've moved this and hopefully gotten the format correct. Thanks